lib/ui/screen/login_screen.dart

```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
-                            Icons.account_circle,
-                            color: Colors.grey,
-                          ),
-                          border: InputBorder.none,
                         ),
                       ),
-                    ),
-                    const SizedBox(
-                      height: 4,
-                    ),
-                    if (_accountErrorMessage.isNotEmpty)
-                      Padding(
-                        padding: const EdgeInsets.only(left: 16),
-                        child: Text(
-                          _accountErrorMessage,
-                          style: const TextStyle(
-                            fontSize: 12,
-                            color: Colors.red,
+                      const SizedBox(
+                        height: 4,
+                      ),
+                      if (_accountErrorMessage.isNotEmpty)
+                        Padding(
+                          padding: const EdgeInsets.only(left: 16),
+                          child: Text(
+                            _accountErrorMessage,
+                            style: const TextStyle(
+                              fontSize: 12,
+                              color: Colors.red,
+                            ),
                           ),
                         ),
+                      const SizedBox(
+                        height: 20,
                       ),
-                    const SizedBox(
-                      height: 20,
-                    ),
-                    Material(
-                      elevation: 2,
-                      borderRadius: const BorderRadius.all(Radius.circular(32)),
-                      child: TextFormField(
-                        controller: _passwordControl,
-                        cursorColor: Colors.blue[800],
-                        obscureText: true,
-                        focusNode: _passwordFocus,
-                        onEditingComplete: () {
-                          _passwordFocus.unfocus();
-                        },
-                        validator: (value) => _validatorPassword(value),
-                        decoration: InputDecoration(
-                          hintText: R.current.password,
-                          errorStyle: const TextStyle(
-                            height: 0,
-                            fontSize: 0,
-                          ),
-                          prefixIcon: const Icon(
-                            Icons.lock,
-                            color: Colors.grey,
-                          ),
-                          border: InputBorder.none,
-                          contentPadding: const EdgeInsets.symmetric(
-                            horizontal: 25,
-                            vertical: 13,
+                      Material(
+                        elevation: 2,
+                        borderRadius: const BorderRadius.all(Radius.circular(32)),
```

Comment:
```suggestion
                        borderRadius: const BorderRadius.circular(32),
```

Reply:
Sure, done in [5539c2b](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/5539c2b6d8b336a021e5f81cabe5d4323107fffb).

Reply:
Done in [9d92f57](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/9d92f57c53a7bda5eebc22af86d0d46c1833ba45).